apiserver/facades/agent/caasoperator/mock_test.go

```diff
@@ -55,8 +55,8 @@ func (st *mockState) Model() (caasoperator.Model, error) {
 	return &st.model, nil
 }
 
-func (st *mockState) APIHostPorts() ([][]network.HostPort, error) {
-	st.MethodCall(st, "APIHostPorts")
+func (st *mockState) APIHostPortsForAgents() ([][]network.HostPort, error) {
+	st.MethodCall(st, "APIHostPortsForAgents")
```

Comment:
it would seem that we never test this functionality. As we're changing the value of a MethodCall for a mock object, but there is no associated test change...